lib/tic_tac_toe.rb

```diff
@@ -18,8 +25,30 @@ def display_welcome
     @user_interface.display_message welcome_message
   end
 
-  def create_a_game
-    game_generator = GameGenerator.new
-    game_generator.create_a_game @user_interface
+  def ask_for_opponent_selection
+    @user_interface.display_empty_line
+    @user_interface.display_message 'Would you like to play a human or a computer?'
+    @user_interface.display_message 'Enter 1 for human, and 2 for computer:'
```

Comment:
all this method does is calling the other method. So maybe asking for opponent selection is not responsibility of TTT class?

Reply:
Also, is there a need to call display_message twice?

Reply:
Agreed, this doesn't feel very Tic Tac Toe specific. Opponents exist in other types of games as well.